packages/testing/src/web/customRender.tsx

```diff
@@ -1,19 +1,17 @@
 import React from 'react'
 
-import { render } from '@testing-library/react'
-import type { RenderResult, RenderOptions } from '@testing-library/react'
+import { render, renderHook } from '@testing-library/react'
 import type {
+  RenderResult,
+  RenderOptions,
   RenderHookOptions,
   RenderHookResult,
-} from '@testing-library/react-hooks'
-// `@testing-library/react-hooks` is being deprecated
-// since the functionality is moving into v13 of `@testing-library/react`.
-// But v13 of `@testing-library/react` drops support for React 17, so we can't upgrade just yet.
-// We can remove `@testing-library/react-hooks` after upgrading Redwood to React 18.
-import { renderHook } from '@testing-library/react-hooks/dom'
+} from '@testing-library/react'
 
 import { MockProviders } from './MockProviders'
 
+export type CustomRenderHookOptions<Props> = RenderHookOptions<Props>
```

Comment:
Why did you need to create the `CustomRenderHookOptions` type? Why not just use `RenderHookOptions`?